ruff.toml

```diff
@@ -2,6 +2,5 @@ line-length = 150
 
```

Comment:
@denik @pietern  why do we have a 150 limit? IMO it defeats the whole purpose of formatting with ruff, we should format using default settings to be compatible with most of setups

Reply:
(let's have this discussion indepdendent from merging this PR)

Reply:
+1 using 150 is non-standard + if we wanted that then we should encode it in all .vscode settings. Let's review separately.